glance/api/v2/images.py

```diff
@@ -126,6 +156,7 @@ def create(self, req, image, extra_properties, tags):
 
         return image
 
+
```

Comment:
added newline, that's imho also wrong as class methods should only have 1 newline between them - only global functions have 2